### PR TITLE
[lenet] Enhance to run LeNet Sample by multi times.

### PR DESCRIPTION
**Pull request by BruceDai**
@huningxin Now visit http://localhost:8080/lenet/?numRuns={n} would support to run LeNet Sample by multi times(n), and to run LeNet Sample once if param 'numRuns' isn't present in the query string, PTAL, thanks.

### CODE REVIEW

File: lenet/main.js [hunk 1]
```diff
@@ -37,6 +37,11 @@ function getInputFromCanvas() {
   return input;
 }
 
+function getMedianValue(arr) {
+  return arr.length % 2 !== 0 ? arr[Math.floor(arr.length / 2)] :
+      (arr[arr.length / 2 - 1] + arr[arr.length / 2]) / 2;
```
Comment (huningxin) [inline lenet/main.js:42]: I suppose the array should be sorted before get the media value, am I correct?

File: lenet/main.js [hunk 2]
```diff
@@ -37,6 +37,11 @@ function getInputFromCanvas() {
   return input;
 }
 
+function getMedianValue(arr) {
```
Comment (huningxin) [inline lenet/main.js:40]: use full name, e.g. `array`

File: lenet/main.js [hunk 3]
```diff
@@ -70,14 +75,42 @@ export async function main() {
...
+
+      if (n < 1) {
+        alert(`The value of param numRuns must be greater than or equal to 1.`);
+        return;
+      }
+
+      let start;
+      let result;
+      let inferenceTime;
+      const inferenceTimeArr = [];
```
Comment (huningxin) [inline lenet/main.js:90]: `inferenceTimeArray`

File: lenet/main.js (hunk 1 above)
Comment (BruceDai) [reply lenet/main.js]: Thanks @huningxin 
It's fixed in new commit, please take another look, thanks.

File: lenet/main.js (hunk 2 above)
Comment (BruceDai) [reply lenet/main.js]: Updated , thanks @huningxin .

File: lenet/main.js (hunk 3 above)
Comment (BruceDai) [reply lenet/main.js]: Updated , thanks @huningxin

**Review by huningxin — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**